lib/task/migrate/migrations/arMigration0170.class.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Add new term to the levels of description taxonomy
+ *
+ * @package    AccesstoMemory
+ * @subpackage migration
+ */
+class arMigration0170
+{
+  const
+    VERSION = 170, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  public function up($configuration)
+  {
+    // Check if term actually exists before adding
+    $criteria = new Criteria;
+    $criteria->add(QubitTerm::TAXONOMY_ID, QubitTaxonomy::DIGITAL_OBJECT_USAGE_ID);
+    $criteria->addJoin(QubitTerm::ID, QubitTermI18n::ID);
+    $criteria->add(QubitTermI18n::NAME, 'External file');
+    $criteria->add(QubitTermI18n::CULTURE, 'en');
+
+    if (null === QubitTerm::get($criteria))
```

Comment:
We're using a fixed id for the term. If it exists, the `QubitMigrate::bumpTerm` should still be called and the id of the existing term changed to the fixed one for this to work properly. Since we don't use `ON UPDATE CASCADE` for foreign keys, it will also require to change the existing relations with that term.

Reply:
Good news, the `QubitTaxonomy::DIGITAL_OBJECT_USAGE_ID` seems to be [a locked taxonomy](https://github.com/artefactual/atom/blob/qa/2.5.x/lib/model/QubitTaxonomy.php#L99), so probably we don't need to check for existence.

Reply:
Nice!